### ISSUE TITLE
Changing the way radio buttons look

### ISSUE THREAD

**Issue by mezerotm** [mezerotm/cmv#51]

**Comment by mezerotm** [comment]
#### TradingView
![image](https://cloud.githubusercontent.com/assets/5396062/26224728/be36e492-3bf1-11e7-9c16-94dc65084fdd.png)
#### cmv
![image](https://cloud.githubusercontent.com/assets/5396062/26224901/8b606178-3bf2-11e7-8ec9-d14679348c75.png)
<hr>
I am trying to get the checkboxes to look like TradingView, and have managed to get the highlighting.
I am unable to hide the radio buttons because if I do then the tooltips added in #46 / 82b5e535aab95128f31a968d36fb02980087ebde go away.

I'm wondering since @NJTuley made this change, would it be easy to disassociate tooltips with radio buttons and instead associate them with the `<div>`  which contains them?
Or maybe you guys like the way it looks now?

**Comment by NJTuley** [comment]
yes you can do that, could we hold off on doing anything with sidebars for the moment though? Right now, in order to make the sidebar content load dynamically like we want in #16 , I am basically doing a Sidebars 2.0 thing. I am going to be making them object-oriented in order to allow for everything in #16 to work and anything else in the future. I would prefer to be able to finish this before making any changes to the sidebars as they stand because any changes made now, I would have to re-implement after the new sidebars are placed in. I should be able to finish the sidebars in a week and a half or so (depending on how much I get scheduled for work), so it should not take that long to do, but I just don't want me or anyone else having to redo code with the new sidebars. As for the reason to do new sidebars, with the new functionality of the sidebars (or rather the expanded functionality), there are becoming too many variables that would need to be in the program, each associated with each sidebar, and it would become very hard to manage and read, let alone work with in the future, to leave it as is.

**Comment by mezerotm** [comment]
Yeah that's fine I can hold off on this, it is an enhancement after all.
